### DISCUSSION TITLE
[Enhancement] [dropdown] dropdown是否可以新增append-to或teleported属性，以指定弹窗内容的安装位置

### DISCUSSION THREAD

**Discussion by shangjingfei** [element-plus/element-plus#8864] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `dropdown`

## Description

在微前端方案中开启子应用的样式隔离，一般的解决方法是在子应用的class名前加一个子应用名的scope，用以限制子应用的样式（包括子应用所引用的element-ui和其它任何css）外溢，而dropdown组件的下拉内容默认是按照在body上的，就会导致加了样式隔离的子应用样式无法应用在body上，希望此组件像popover一样可以指定安装节点。Thanks♪(･ω･)ﾉ

### Additional comments

_(empty)_

**Comment by tolking** [comment]
从 2.2.20 开始就支持 teleported 属性
